src/pages/MyTokens/NFTPage.tsx

```diff
@@ -136,13 +138,13 @@ export const NFTPage = () => {
       };
 
       let filteredByPrice = true;
-      if (prices?.minPrice && prices?.maxPrice) {
+      if (prices?.minPrice || prices?.maxPrice) {
         if (!token.price) {
           filteredByPrice = false;
         } else {
           const tokenPrice = new BN(token.price);
-          const minPrice = new BN(fromStringToBnString(prices.minPrice, api?.market?.kusamaDecimals));
-          const maxPrice = new BN(fromStringToBnString(prices.maxPrice, api?.market?.kusamaDecimals));
+          const minPrice = new BN(fromStringToBnString(prices.minPrice || '0', api?.market?.kusamaDecimals));
+          const maxPrice = prices.maxPrice ? new BN(fromStringToBnString(prices.maxPrice, api?.market?.kusamaDecimals)) : BN_MAX_INTEGER;
```

Comment:
better not to send that field in request and let back-end resolve it. Force putting max_int is a bad practice.

Reply:
There are no backend requests. These filters are only used on the frontend.